src/amge/DofHandler.cpp

```diff
@@ -125,11 +125,18 @@ const SparseMatrix & DofHandler::GetrDofDofTable(entity type)
         for (int kk(0); kk < nnz+1; ++kk)
             i_Rdof_dof[kk] = kk;
 
+        // XXX: This seems a bit weird. Not only because of the disrespect to
```

Comment:
Maybe say more specifically what you would like to see different, even if we don't take time to fix it all now. (I don't think this comment as written now really helps future readers of the code.)

Reply:
I actually fixed the constness. Turned out easy. The const_cast is still there to satisfy the interface in MFEM but the const-correctness is maintained. Explained in a comment in the code.